charts/endeavor/values.yaml

```diff
@@ -0,0 +1,164 @@
+environment: "production"
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: rotationalio/endeavor
+  # This sets the pull policy for images.
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# This is for the secrets for pulling an image from a private repository more information can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
+
+# Set the Endeavor specific configuration.
+endeavor:
+  # Maintenance mode can be used to run the pod without starting the serice for debugging
+  maintenance: false
+  # Specify the mode of the server (release, debug, testing)
+  mode: release
+  # Specify the verbosity of logging (trace, debug, info, warn, error, fatal, panic)
+  logLevel: info
+  # If true logs colorized human readable output instead of json
+  consoleLog: true
+  # DSN containing backend database configuration
+  databaseURL: sqlite3:///endeavor.db
+  # DSN containing backend database configuration for documents
+  documentsURL: clover:///documents.db
+  # Web server specific configuration
+  web:
+    # If true sets the web UI to run in maintenance mode; inherited from parent
+    maintenance: false
+    # If false, the web UI server will not be run
+    enabled: true
+    # If false, the API server will return unavailable when accessed; subordinate to the enabled flag
+    APIenabled: true
+    # If false, the UI server will return unavailable when accessed; subordinate to the enabled flag
+    UIenabled: true
+    # The ip address and port to bind the web server on
+    bindAddr: 8000
+    # Origin (url) of the UI server for creating endpoints and CORS access
+    origin: http://localhost:8000
+    # Auth Manages authentication and authorization for the Endeavor API server.
+    auth:
+      # Static key configuration as a map of keyID to path on disk
+      keys: {}
+      # The value of the audience JWT claim
+      audience: http://localhost:8000
+      # The value of the issuer JWT claim
+      issuer: http://localhost:8000
+      # Limit cookies to the specified domain (exclude port)
+      cookieDomain: localhost
+      # The amount of time before an access token expires
+      accessTokenTTL: 1h
+      # The amount of time before a refresh token expires
+      refreshTokenTTL: 2h
+      # The amount of overlap between the access and refresh tokens
+      tokenOverlap: -15m
+    docsName: ""
+  # Inference specifies the configuration for the inference client
+  inference:
+    # The URL of the inference endpoint
+    endpointURL: http://localhost:8000
+    # The api key to use for the inference client, if required
+    apikey: apikey
```

Comment:
Maybe there should be a `fromSecret` or `secret` option to include it from an existing secret here, so the values.yaml would have to include the secret name and key name here.